bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "sbvr-types",
```

Comment:
You can kill this file altogether now - bower itself recommends switching to node_modules and one of the many systems that work well with node_modules